test/events/event_factory.js

```diff
@@ -55,26 +55,27 @@ contract('EventFactory', function(accounts) {
                 await EventFactory.new(0, { from: eventFactoryCreator });
                 assert.fail();
             } catch(e) {
-                assertInvalidOpcode(e);
+                SolAssert.assertRevert(e);
             }
         });
     });
 
     describe('TopicEvent:', async function() {
         it('initializes all the values of the new topic correctly', async function() {
             assert.equal(await topic.owner.call(), topicCreator);
-            assert.equal(await topic.getEventName(), testTopicParams._name.join(''));
+            assert.equal(web3.toUtf8(await topic.name.call(0)), testTopicParams._name[0]);
+            assert.equal(web3.toUtf8(await topic.name.call(1)), testTopicParams._name[1]);
             assert.equal(web3.toUtf8(await topic.resultNames.call(0)), testTopicParams._resultNames[0]);
             assert.equal(web3.toUtf8(await topic.resultNames.call(1)), testTopicParams._resultNames[1]);
             assert.equal(web3.toUtf8(await topic.resultNames.call(2)), testTopicParams._resultNames[2]);
             assert.equal((await topic.numOfResults.call()).toNumber(), 3);
 
-            let centralizedOracle = await CentralizedOracle.at((await topic.getOracle(0))[0]);
-            assert.equal(await centralizedOracle.oracle.call(), testTopicParams._oracle);
-            assert.equal(await centralizedOracle.getEventName(), testTopicParams._name.join(''));
-            assert.equal(await centralizedOracle.getEventResultName(0), testTopicParams._resultNames[0]);
-            assert.equal(await centralizedOracle.getEventResultName(1), testTopicParams._resultNames[1]);
-            assert.equal(await centralizedOracle.getEventResultName(2), testTopicParams._resultNames[2]);
+            let centralizedOracle = await CentralizedOracle.at((await topic.oracles.call(0))[0]);
+            assert.equal(web3.toUtf8(await centralizedOracle.eventName.call(0)), testTopicParams._name[0]);
+            assert.equal(web3.toUtf8(await centralizedOracle.eventName.call(1)), testTopicParams._name[1]);
+            assert.equal(web3.toUtf8(await centralizedOracle.eventResultNames.call(0)), testTopicParams._resultNames[0]);
+            assert.equal(web3.toUtf8(await centralizedOracle.eventResultNames.call(1)), testTopicParams._resultNames[1]);
+            assert.equal(web3.toUtf8(await centralizedOracle.eventResultNames.call(2)), testTopicParams._resultNames[2]);
             assert.equal(await centralizedOracle.numOfResults.call(), 3);
```

Comment:
Out of curiosity, the reason for those syntax change?

Reply:
I removed the getters that I defined, `getEventResultNames()` and just made the `resultNames` var public. This auto creates getters thus making the user defined `getEventResultNames()` unnecessary. The way to call public vars in JS is `varName.call()`